aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -881,11 +880,11 @@ impl AptosVM {
 
         // The check below should have been feature-gated in 1.11...
         if function.is_friend_or_private() {
-            let metadata = module_storage.fetch_existing_module_metadata(
-                entry_fn.module().address(),
-                entry_fn.module().name(),
-            )?;
-            if get_randomness_annotation_for_entry_function(entry_fn, &metadata).is_some() {
+            let maybe_randomness_annotation = get_randomness_annotation_for_entry_function(
+                entry_fn,
+                &function.expect_module()?.metadata,
```

Comment:
owner_as_module? or maybe we can find a better name for both, but to avoid expect

Reply:
Renamed to `owner_as_module`